apps/totp-authentication-portal/src/main/webapp/totp.jsp

```diff
@@ -68,6 +68,8 @@
             <jsp:include page="includes/header.jsp"/>
             <% } %>
 
+            <script src="libs/jquery_3.6.0/jquery-3.6.0.js"></script>
```

Comment:
How about adding these couple of scripts to includes/header.jsp?

Reply:
There are no scripts included in the header.jsp files except for one occurrence (in authetication-portal) in the repository. Wouldn't it be better for consistency to not change the maintained pattern?